src/components/shared/description-list/description-list.tsx

```diff
@@ -18,7 +19,7 @@ const DescriptionList: React.FC<DescriptionListProps> = ({ title, items }) => {
       title={title}
       headerRight={<></>}
       list={items.map((b) => (
-        <>{b.detail}</>
+        <Fragment>{b.detail}</Fragment>
```

Comment:
```suggestion
        <>{b.detail}</>
```
shorthand syntax FTW ;p